test/generator/filler/test_runner.py

```diff
@@ -197,6 +198,7 @@ def test_add_relative_hint(
         )
     else:
         max_distance = 20
+        precision = HintItemPrecision.GENERAL_CATEGORY
```

Comment:
I guess the type error comes from the fact that mypy infers the type from the first usage.

Reply:
oh these warnings are good catch that I forgot to add these files as required for mypy